vmm/src/api/http/http_endpoint.rs

```diff
@@ -220,6 +219,34 @@ impl PutHandler for VmAddNet {
 
 impl GetHandler for VmAddNet {}
 
+impl PutHandler for VmRestore {
```

Comment:
I think it makes sense to implement a convenience macro here because VmAddNet PutHandler is implementing the same logic that you have implement for VmRestore.

Reply:
Thanks for the suggestion. Added a new macro - `vm_action_put_handler_body_with_fds`